### ISSUE TITLE
Inconsitency in filter option

### ISSUE THREAD

**Issue by sarmong** [kyazdani42/nvim-tree.lua#857]
`filter` option currently includes two options - `dotfiles` and `custom`. 

It seems a bit inconsistent that dotfiles are toggled with `tree_cb("toggle_dotfiles")` function, but `custom` are toggled with `tree_cb("toggle_ignored") `.

Not sure what would be better way to handle this, as toggling `custom` with `toggle_dotfiles` is not the best way either.

**Comment by kyazdani42** [comment]
maybe toggle_custom_filter ?

**Comment by alex-courtis** [comment]
```
H : toggle_dotfiles
I : toggle_git_ignored
U : toggle_custom
```

`toggle_ignored` would be replaced with `toggle_git_ignored`, with `toggle_ignored` legacy silently mapped.

I think the main use case for `I` is git ignored, as per nerdtree and others, hence we retain that and create the new `U`.

**Comment by kyazdani42** [comment]
closing since implemented in #1077 :)
